tests/manage/pv_services/space_reclaim/test_auto_reclaim_space_cronjob.py

```diff
@@ -0,0 +1,228 @@
+import logging
+import pytest
+import yaml
+from uuid import uuid4
+import os
+from tempfile import NamedTemporaryFile
+
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    ManageTest,
+    tier2,
+)
+from ocs_ci.utility.utils import run_cmd, TimeoutSampler
+from ocs_ci.helpers.performance_lib import run_oc_command
+from ocs_ci.helpers import helpers, performance_lib
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import PVCNotCreated, TimeoutExpiredError
+
+logger = logging.getLogger(__name__)
+ERRMSG = "Error in command"
+
+
+@tier2
+@skipif_ocs_version("<4.14")
+class TestReclaimSpaceCronJob(ManageTest):
+    """
+    Test that verifies automatic creation of Reclaim Space Cron Jobs for RBD PVCs in openshift-* namespaces
+    The test also verifies that no Reclaim Space Cron Job is created automatically for CephFS PVC in this namespace
+    """
+
+    def setup(self):
+        self.pvc_objs_created = []
+        self.temp_files_list = []
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request):
+        """
+        Cleanup the test environment
+
+        """
+
+        def finalizer():
+            # pvc objects usually are deleted in the test (and the second delete has no effect),
+            # if for some reason the pvcs were not deleted in the test - they will be deleted here
+            for pvc_obj in self.pvc_objs_created:
+                pvc_obj.delete()
+
+            for temp_file in self.temp_files_list:
+                if os.path.exists(temp_file):
+                    run_cmd(f"rm {temp_file}")
+
+            run_oc_command(cmd=f"delete namespace {self.namespace}")
+
+        request.addfinalizer(finalizer)
+
+    def test_reclaim_space_cronjob(self):
+        """
+        Test case to check reclaim space cronjobs are created correctly for rbd pvcs in openshift-* namespace
+        """
+        num_of_pvcs = 10
+        namespace = f"openshift-{uuid4().hex}"
+        self.namespace = namespace
+        result = run_oc_command(cmd=f"create namespace {self.namespace}")
+        assert ERRMSG not in result[0], (
+            f"Failed to create namespace with name {namespace}" f"got result: {result}"
+        )
```

Comment:
Is some error expected here ? If yes, plese uuse try: except

Reply:
There is no exception thrown when oc commands runs, even in the case of failure. If the namespace  failed to be created, the error will be seen in the 'result' and this is exactly what I am checking.